pkg/api/annotations.go

```diff
@@ -54,12 +57,20 @@ func (e *CreateAnnotationError) Error() string {
 	return e.message
 }
 
-func PostAnnotation(c *models.ReqContext) response.Response {
+func (hs *HTTPServer) PostAnnotation(c *models.ReqContext) response.Response {
 	cmd := dtos.PostAnnotationsCmd{}
 	if err := web.Bind(c.Req, &cmd); err != nil {
 		return response.Error(http.StatusBadRequest, "bad request data", err)
 	}
-	if canSave, err := canSaveByDashboardID(c, cmd.DashboardId); err != nil || !canSave {
+
+	var canSave bool
```

Comment:
I like this improvement and stop using names like "saveByDashboard" and start talking about annotations local and global types. `canSave` was being used as default `true` so to keep the same behavior we need to declare it as `true` because in case of `global` annotation and FGAC enabled, canSave will be always false with this implementation.

```suggestion
	canSave := true
```

Reply:
You're very right, thanks for catching this! I've fixed it now.